### DISCUSSION TITLE
'exit'-Keyword

### DISCUSSION THREAD

**Discussion by PhilippElhaus** [dotnet/csharplang#7314] [Language Ideas]
Add the `exit`-keyword that breaks all current loops.

The Problem: 
Sometimes you are in nested loops 2+ levels deep and need to terminate them all. Currently this can only be achieved by complex code with labeled `break` keywords or an overkill `return` that ends the entire method. 

The Solution:
The new `exit`keyword. It terminates all current loop structures and continues at the root of the executing method.

Example:

```
public void ExitExample()
{
    
    for (int i = 1; i <= 3; i++)
    {
      
        int j = 1;
        while (j <= 2)
        {
    
            for (int k = 1; k <= 3; k++)
            {
                  exit;
            }

            j++;
        }
    }
   
// Code continues here after exit 

}

```

**Comment by TahirAhmadov** [comment]
https://github.com/dotnet/csharplang/discussions/6634

**Comment by jnm2** [comment]
`goto` is what I use in a scenario like this, and I'm happy to see others using it as well. The rumored dangers of `goto` are vastly exaggerated in modern languages like C#. This is particularly easy to see when the `goto` lands immediately before or after the start or end of a control-flow construct, where it's exactly equivalent to `break 2;` or `continue 2;` or equivalent to an outer flag-conditional loop.
